connector/src/main/java/org/geysermc/connector/utils/LocaleUtils.java

```diff
@@ -44,13 +49,18 @@
 
     private static final String DEFAULT_LOCALE = (GeyserConnector.getInstance().getConfig().getDefaultLocale() != null ? GeyserConnector.getInstance().getConfig().getDefaultLocale() : "en_us");
 
+    private static String smallestURL = "";
+
     static {
         // Create the locales folder
         File localesFolder = new File("locales/");
         localesFolder.mkdir();
 
         // Download the latest asset list and cache it
         generateAssetCache();
+        if (!DEFAULT_LOCALE.equals("en_us")) {
```

Comment:
So are we always downloading "en_us"? Feels like this should be the other way around.

Reply:
I thought it would be best, so when an en_us client joins they don't get language strings if they request a message with one before the server has downloaded it.

Reply:
There will be some Geyser users that never touch `en_us`, although for the majority this does apply, I assume. Could we consider sending a message to them saying that their locale will be downloaded shortly?

Reply:
Pushed the change to stop auto-download of en_us still need to let the user know.